discovery-provider/src/tasks/tracks.py

```diff
@@ -117,6 +117,26 @@ def invalidate_old_track(session, track_id):
     ), "Update operation requires a current track to be invalidated"
 
 
+def update_credit_splits(session, track_record, track_metadata):
+    child_track_id = track_record.track_id
+
+    # Delete existing credits
+    session.query(Credit).filter_by(child_track_id=child_track_id).delete()
+
+    # Add all credits
+    if "credits_splits" in track_metadata:
+        splits = track_metadata["credits_splits"].get("splits")
+        if splits:
+            for split in splits:
+                parent_track_id = split.get("track_id")
+                if parent_track_id:
+                    credit = Credit(
```

Comment:
for example, if a split is removed in an update is that previous credit entry still present in the database?

Reply:
@hareeshnagaraj  for your context, we resolved this on VC this morning.

Based on our decisions & this code here:
1. the "credits splits" table which we're renaming to "remixes" is purely for querying aide, *not* for representing chain state. All of the rows for a given track id are cleared whenever the credit splits are updated in the track metadata.
2. We don't need block number because the uni-directional relationship between a child and a parent is captured in the tracks table (which is indexed for notifications).

Lmk if you wanna chat more about it

Reply:
awesome! yes, with 1. this is basically fixed since whatever is in the table is always going to be updated. looks like i missed that on line 124 actually so we should be good